### ISSUE TITLE
Post Processing Effects

### ISSUE THREAD

**Issue by Cikatriz87** [p1xel8ted/UltrawideFixes#23]
![Screenshot 2024-05-28 154649](https://github.com/p1xel8ted/UltrawideFixes/assets/41704518/ec31a968-5c0c-45e5-9734-8ac7447cacb0)
Hi. It seems the post processing effects aren't loading. They are all off by default and cannot be enabled

**Comment by p1xel8ted** [comment]
You using the retail full release? Not the demo

**Comment by Cikatriz87** [comment]
> You using the retail full release? Not the demo

Yes, but I think I managed to get it to work by adding Makotech's mod then this one. I can only see the default resolution when I disable all the effects though (shown in screenshot). That's okay since I plan to play with the effects on  but  would be nice to up the default resolution with the effects enabled
![Screenshot 2024-05-28 162622](https://github.com/p1xel8ted/UltrawideFixes/assets/41704518/8db1a506-49af-49f2-aff1-1bb4208750d0)

**Comment by Cikatriz87** [comment]
I tried changing the inernal resolution multiplier via config file and didn't work. Here's a  shot with the effects on and default res. I probably shouldn't be using both mods but it's working okay for me so far
![Screenshot 2024-05-28 163140](https://github.com/p1xel8ted/UltrawideFixes/assets/41704518/c901fb47-e843-4fe1-9e50-3cb5ee4fbb2b)

**Comment by p1xel8ted** [comment]
The update on the 24th May broke it.

**Comment by p1xel8ted** [comment]
Fixed in 0.1.1
https://github.com/p1xel8ted/UltrawideFixes/releases/tag/CrowCountry
